README.md

```diff
@@ -6,7 +6,7 @@
 [![Bugs](https://sonarcloud.io/api/project_badges/measure?project=cosmos_interchain-security&metric=bugs)](https://sonarcloud.io/summary/new_code?id=cosmos_interchain-security)
 [![Lines of Code](https://sonarcloud.io/api/project_badges/measure?project=cosmos_interchain-security&metric=ncloc)](https://sonarcloud.io/summary/new_code?id=cosmos_interchain-security)
 
-**interchain-security** houses the code for implementing Interchain Security: https://blog.cosmos.network/interchain-security-is-coming-to-the-cosmos-hub-f144c45fb035. The repo is currently a WIP and targetting v1 of Interchain Security.
+**interchain-security** houses the code for implementing [Interchain Security](https://blog.cosmos.network/interchain-security-is-coming-to-the-cosmos-hub-f144c45fb035) ([blog post](https://blog.cosmos.network/interchain-security-is-coming-to-the-cosmos-hub-f144c45fb035)). The repo is currently a WIP and targetting v1 of Interchain Security.
```

Comment:
There are two links to the same blog post.

Reply:
Yeah I know - should there be a different link? Having two links is nice from readability perspective, people don't link to click random stuff. That's why I added the '(blog post)' even though it's the same link.

Reply:
```suggestion
**interchain-security** houses the code for implementing Interchain Security (for more details, take a look at this [blog post](https://blog.cosmos.network/interchain-security-is-coming-to-the-cosmos-hub-f144c45fb035)). The repo is currently a WIP and targetting v1 of Interchain Security.
```